source/sass/blocks/footer-social.scss

```
@@ -35,7 +35,7 @@
   background-repeat: no-repeat;
   background-position: center;
   color: $cerise;
-  border: 3px solid $midnight;
+  border: 3px solid rgba(40, 54, 69, 0.1);
```

А ты знаешь, что благодаря препроцессору можно делать так:
```suggestion
  border: 3px solid rgba($midnight, 0.1);
```

---

не знала
исправила